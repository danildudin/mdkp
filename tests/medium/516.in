30 9 0
75 469 166 749 677 717 492 458 180 527 57 691 934 366 470 644 729 412 737 365 36 875 189 109 750 995 910 994 117 487 
983 344 142 447 665 776 848 510 370 749 703 696 861 868 398 978 397 416 390 819 504 665 953 365 960 768 964 248 997 920 
741 258 268 373 962 890 879 557 587 479 161 805 610 876 233 94 945 252 860 57 411 866 1 548 833 912 241 726 205 220 
220 566 87 226 750 25 22 615 618 34 287 930 108 164 615 638 401 493 892 506 523 43 807 288 388 353 723 122 749 468 
444 176 991 314 617 626 82 299 79 228 785 475 850 43 617 857 694 165 593 953 663 135 963 928 230 807 29 727 397 919 
203 720 726 147 45 347 917 382 154 820 397 52 766 845 295 353 833 338 328 751 692 861 176 153 87 319 318 748 675 222 
850 277 99 385 961 928 961 45 722 997 538 740 658 460 639 495 905 1 230 584 961 637 84 167 915 701 500 863 698 204 
212 764 379 894 91 125 63 16 749 463 195 591 400 426 442 642 14 847 220 848 562 740 926 631 601 69 764 783 981 506 
585 2 652 671 336 935 900 681 728 154 570 638 64 721 67 928 699 236 511 44 146 776 784 340 714 769 821 110 471 670 
958 619 161 204 683 960 513 142 303 845 721 486 846 918 429 930 432 203 867 692 454 685 187 481 405 808 156 305 377 211 
6717 6700 9822 7851 6650 9270 8046 8963 8631 
