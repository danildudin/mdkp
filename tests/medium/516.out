8694
12
1 3 7 11 12 14 17 18 21 25 26 27 
